node/resolvers/auth/index.ts

```diff
@@ -24,5 +24,6 @@ export const mutations = {
     const { fields: { email, authToken, code } } = args
     const { data: { authCookie } } = await makeRequest(ioContext, paths.signIn(email, authToken, code))
     response.set('Set-Cookie', serialize(authCookie.Name, authCookie.Value, { httpOnly: true }))
+    return authCookie
```

Comment:
Why are you returning the AutheCookie if you already set it? Does the UI need this (sensitive) value for anything?

Reply:
This is just a boolean return. I'm not passing the authToken

Check here: https://github.com/vtex-apps/store-graphql/pull/42/files#diff-045015134d7ea6d666b89fb301e37305R60